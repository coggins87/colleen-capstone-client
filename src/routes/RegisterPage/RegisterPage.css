.RegisterPage {
  display: flex;
  flex-direction: column;
  margin: auto;
  padding-top: 9%;
}

@media all and (max-width: 500px){
  .RegisterPage{
    padding-top: 25%
  }
}