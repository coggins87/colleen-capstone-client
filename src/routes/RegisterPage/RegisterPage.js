import React from 'react'
import RegisterForm from '../../components/RegisterForm/RegisterForm'

class RegisterPage extends React.Component {
 
  render(){
    return(
      <section>
        <h2>Register</h2>
     <RegisterForm />
     </section>
    )
  }
}

export default RegisterPage