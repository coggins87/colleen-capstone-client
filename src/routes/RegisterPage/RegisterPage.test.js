import React from 'react';
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import RegisterPage from './RegisterPage'

describe(`RegisterPage component`, () => {
const props = {
  history: {
    push:()=>{}
  },
}

  it('renders the RegisterPage component by default', () => {
    const wrapper = shallow(<RegisterPage />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })
  it('renders the RegisterPage component given props', () => {
    const wrapper = shallow(<RegisterPage {...props}/>)
    expect(toJson(wrapper)).toMatchSnapshot()
  })


})