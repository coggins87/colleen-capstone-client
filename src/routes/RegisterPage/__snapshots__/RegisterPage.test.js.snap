// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RegisterPage component renders the RegisterPage component by default 1`] = `
<section
  className="RegisterPage"
>
  <h2>
    Register
  </h2>
  <RegisterForm
    onRegistrationSuccess={[Function]}
  />
</section>
`;

exports[`RegisterPage component renders the RegisterPage component given props 1`] = `
<section
  className="RegisterPage"
>
  <h2>
    Register
  </h2>
  <RegisterForm
    onRegistrationSuccess={[Function]}
  />
</section>
`;
